dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12-bookworm as base
```

Comment:
Minor, but I recommend trying to call this file `Dockerfile` as opposed to `dockerfile` - _most_ systems will be forgiving and it won't really matter, but that almost just makes it more confusing when it occasionally fails because it was expecting a capital D

You may find Git ignores the file name change, and you need to use the `git mv` command (as per [this SO post](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git)) to get Git to notice any changes